pkg/volume/glusterfs/glusterfs.go

```diff
@@ -671,6 +680,10 @@ func parseClassParameters(params map[string]string, kubeClient clientset.Interfa
 		return nil, fmt.Errorf("StorageClass for provisioner %s must contain 'resturl' parameter", glusterfsPluginName)
 	}
 
+	if cfg.gid == 0 {
+		cfg.gid = defaultGid
+	}
```

Comment:
@obnoxxx thought of keeping the defaultGid const for below reasons, 
with its presence its clear to whoever read the code and also easy to change it if we need to.